ivy/container/sorting.py

```diff
@@ -48,6 +128,52 @@ def argsort(
         *,
         out: Optional[ivy.Container] = None
     ) -> ivy.Container:
+        """
+        ivy.Container instance method variant of ivy.argsort. This method simply wraps the
+        function, and so the docstring for ivy.argsort also applies to this method
+        with minimal changes.
+        
+        Parameters
+        ----------
+        self 
+            input array or container. Should have a numeric data type.
+        axis
+            axis along which to sort. If set to ``-1``, the function must sort along the last axis. Default: ``-1``.
+        descending
+            sort order. If ``True``, the returned indices sort ``x`` in descending order (by value). If ``False``, the returned indices sort ``x`` in ascending order (by value). Default: ``False``.
+        stable
+            sort stability. If ``True``, the returned indices must maintain the relative order of ``x`` values which compare as equal. If ``False``, the returned indices may or may not maintain the relative order of ``x`` values which compare as equal (i.e., the relative order of ``x`` values which compare as equal is implementation-dependent). Default: ``True``.
```

Comment:
this line is too long, please update it so that one line only has up to 88 characters, rest of the content would go to a new line.